webpack/scenes/ContentViews/Details/Versions/Delete/RemoveSteps/CVVersionRemoveReview.js

```diff
@@ -53,20 +64,52 @@ const CVVersionRemoveReview = () => {
       {affectedHosts &&
         <>
           <h3>{__('Content hosts')}</h3>
-          <Flex>
-            <FlexItem><ExclamationTriangleIcon /></FlexItem>
-            <FlexItem><p>{__(`${pluralize(hostResponse.length, 'host')} will be moved to content view ${selectedCVNameForHosts} in `)}</p></FlexItem>
-            <FlexItem><Label isTruncated color="purple" href={`/lifecycle_environments/${selectedEnvForHost[0].id}`}>{selectedEnvForHost[0].name}</Label></FlexItem>
-          </Flex>
+          {singleCVHostsCount > 0 && (
+            <Flex>
+              <FlexItem><ExclamationTriangleIcon /></FlexItem>
+              <FlexItem><p>{__(`${pluralize(singleCVHostsCount, 'host')} will be moved to content view ${selectedCVNameForHosts} in `)}</p></FlexItem>
+              <FlexItem><Label isTruncated color="purple" href={`/lifecycle_environments/${selectedEnvForHost[0].id}`}>{selectedEnvForHost[0].name}</Label></FlexItem>
+            </Flex>
+          )}
+          {multiCVHostsCount > 0 && (
+            <Flex>
+              <FlexItem><ExclamationTriangleIcon /></FlexItem>
+              <FlexItem>
+                <p>
+                  {__(`Content view environment 
+                    ${selectedEnv
+                    ?.map(({ name }) => `${name}/${multiCVHosts[0]?.content_facet_attributes?.content_view?.name}`)
+                    .join(', ')} 
+                    will be removed from ${pluralize(multiCVHostsCount, 'multi-environment host')}.`)}
```

Comment:
When using `${...}`, the `__()` won't properly translate the string. We should use something similar to this:
https://github.com/Katello/katello/blob/527df0a5da5379d19ade2a7b3abcbbf43fc5b72b/webpack/scenes/Hosts/ChangeContentSource/components/ContentSourceForm.js#L248-L264